app/players/__init__.py

```diff
@@ -0,0 +1,6 @@
+def elvese() -> None:
+    return None
+
+
+def elveselves() -> None:
+    return None
```

Comment:
The functions `elvese` and `elveselves` are not required for the current task. Consider removing them to keep the codebase clean and focused on the task requirements.